app/bundles/CoreBundle/Helper/BuilderTokenHelper.php

```diff
@@ -73,7 +73,7 @@ public function getTokenContent($page = 1, $entityArguments = array(), $viewPara
         );
 
 
-        if (in_array(false, $permissions)) {
+        if (count(array_unique($permissions)) == 1 && end($permissions) == false) {
```

Comment:
Not against this code block but just curious the thought behind it vs something like `if (!in_array(true, $permissions)`?